code/modules/pai/software.dm

```diff
@@ -135,16 +135,18 @@
  */
 /mob/living/silicon/pai/proc/change_image()
 	var/list/possible_choices = list()
-	for(var/face_option in possible_overlays)
+	for(var/path in subtypesof(/datum/pai_screen_image))
+		var/datum/pai_screen_image/screen_option = path
 		var/datum/radial_menu_choice/choice = new
-		choice.name = face_option
-		choice.image = image(icon = card.icon, icon_state = "pai-[face_option]")
-		possible_choices[face_option] += choice
+		choice.name = screen_option.name
+		choice.image = image(icon = screen_option.icon, icon_state = screen_option.icon_state)
+		possible_choices[screen_option] += choice
```

Comment:
```suggestion
		choice.name = initial(screen_option.name)
		choice.image = image(icon = initial(screen_option.icon), icon_state = initial(screen_option.icon_state))
		possible_choices[screen_option] += choice
```

Was this tested? `subtypes()` returns a list of abstract paths so directly referencing the name or any other attribute without `initial()` will runtime.

Reply:
It was tested and worked fine. Do you still want me to add `initial()` around everything?

Reply:
It's not required then